code/WorkInProgress/warcrimes.dm

```diff
@@ -28,7 +28,6 @@ TYPEINFO(/area/diner/juicer_trader)
 
 /obj/item/clothing/shoes/thong
 	name = "garbage flip-flops"
-	desc = "These cheap sandals don't even look legal."
```

Comment:
why rm

Reply:
Fixes the double description from #20149 
Unless there is a nother way to do it.